tests/modules/moduleparameter.c

```diff
@@ -0,0 +1,25 @@
+#include "valkeymodule.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+int GET_HELLO(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
+  ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);
+  return ValkeyModule_ReplyWithSimpleString(ctx, "Module runtime args test");
+}
+
+int ValkeyModule_OnLoad(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
+    VALKEYMODULE_NOT_USED(argv);
+    VALKEYMODULE_NOT_USED(argc);
+
+    if (ValkeyModule_Init(ctx,"myhello",1,VALKEYMODULE_APIVER_1)
+        == VALKEYMODULE_ERR) return VALKEYMODULE_ERR;
+
+
+    if (ValkeyModule_CreateCommand(ctx,"hello.hi",
+        GET_HELLO,"fast",0,0,0) == VALKEYMODULE_ERR)
```

Comment:
Command name "hello.hi" and function name GET_HELLO are not perfect names. :)

Something like "updateargs", updateArgsCommand?

Reply:
Thanks for your comments, it is updated